app/views/users/index.html.erb

```diff
@@ -1,12 +1,34 @@
 <section class="users-section">
   <ul class="users-list">
-    <% @users.each do |user| %>
-      <li>
-        Name: <%= user.name %>
-        <span class="profile-link">
-          <%= link_to 'See Profile',  user_path(user), class: 'profile-link' %>
-        </span>
-      </li>
-    <% end %>
+<table>
+<tr>
+  <th>User</th>
+  <th>See profile</th>
+  <th colspan="2">Action</th>
+</tr>
+      <% @users.each do |user| %>
+      <% if current_user != user %>
+        <tr>
+            <td>
+              <%= user.name %>
+              <span class="profile-link">
+            </td>
+              <td><%= link_to 'See Profile',  user_path(user), class: 'profile-link' %></td>
+            
+              <% if current_user.friend?(user) %>
+                <td><%= link_to '<button type="button">Unfriend</button>'.html_safe, friendship_path(user), method: :delete, data: { confirm: "Are you sure you want to Unfriend"} %></td>
```

Comment:
- [ ] Let's move this whole logic block foe the friendship buttons into a helper method. Remember to avoid view logic directly at views.